tools/github/data/common-labels.csv

```diff
@@ -1,6 +1,7 @@
 AAD,,e99695
 ACS,,e99695
 AI Model Inference,,e99695
+AI Projects,,e99695
```

Comment:
curious: is this like a generic label for all things AI? or internal projects?
how will it be use?

Reply:
No, It's a new offering from the AI group. The offering is called AI Projects. You can use it to create AI agents.

Reply:
🫣🧐☺️